tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -54,6 +54,12 @@ internal static partial class ConfigurationKeys
         /// </summary>
         public const string DebugEnabled = "DD_TRACE_DEBUG";
 
+        /// <summary>
+        /// Configuration key for enabling or disabling the Tracer's debugger mode.
+        /// Default is value is false (disabled).
+        /// </summary>
+        public const string DebuggerEnabled = "DD_TRACE_DEBUGGER";
```

Comment:
```suggestion
        public const string DebuggerEnabled = "DD_INTERNAL_WAIT_FOR_DEBUGGER_ATTACH";
```

Reply:
We should avoid conflicting with names that might be used by the Live Debugging team in the future

Reply:
Maybe in the future we should also block in native when this env var is enabled (or have a separate one for the native 🤔 ).

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3613/commits/2e5c371193338f4585b8ff83821cc2e9edcde605